src/legacy/core_plugins/data/public/search/aggs/param_types/field.test.ts

```diff
@@ -125,20 +120,9 @@ describe('Field', () => {
       const aggParam = new FieldParamType({
         name: 'field',
         type: 'field',
-        filterFieldTypes: (aggConfig: IMetricAggConfig) =>
-          get(aggConfig.schema, 'aggSettings.top_hits.allowStrings', false)
-            ? '*'
-            : KBN_FIELD_TYPES.NUMBER,
+        filterFieldTypes: (aggConfig: IMetricAggConfig) => '*',
```

Comment:
It's a `*` by default
```suggestion
```

Reply:
oh sorry,
I just see that these are in tests file,
anyway I mean, that `filterFieldTypes` as a function became redundant,
so you could even remove it from tests